devops/cloudformation.json

```diff
@@ -0,0 +1,149 @@
+{
+  "Parameters": {
+    "RREHostname": {
+      "Type": "String",
+      "Default": "rre-beta.labs.transitmatters.org",
+      "AllowedPattern": "^regionalrail\\.rocks$|^.*\\.labs\\.transitmatters\\.org$",
+      "Description": "The hostname for the site; for example: regionalrail.rocks."
+    },
+    "RREDomain": {
+      "Type": "String",
+      "Default": "labs.transitmatters.org",
+      "AllowedPattern": "^regionalrail\\.rocks$|^labs\\.transitmatters\\.org$",
+      "Description": "The parent domain of the site. Most likely regionalrail.rocks or labs.transitmatters.org."
+    },
+    "RRECertArn": {
+      "Type": "String",
+      "Description": "The ACM ARN of the SSL certificate."
+    }
+  },
+  "Resources": {
+    "RREInstance": {
+      "Type": "AWS::EC2::Instance",
+      "Properties": {
+        "ImageId": "ami-070650c005cce4203",
+        "InstanceType": "t4g.small",
+        "KeyName": "transitmatters-rre",
+        "SecurityGroupIds": [
+          { "Ref": "RRESecurityGroup" }
+        ],
+        "Tags": [
+          {
+            "Key": "Name",
+            "Value": {"Fn::Sub": "rre-${RREHostname}"}
+         }
+        ]
+      }
+    },
+    "RRESecurityGroup": {
+      "Type": "AWS::EC2::SecurityGroup",
+      "Properties": {
+        "GroupName": {"Fn::Sub": "RRE-securitygroup-${RREHostname}"},
+        "GroupDescription": "Allow HTTP/HTTPS and SSH inbound and outbound traffic",
+        "SecurityGroupIngress": [
+          {
+            "IpProtocol": "tcp",
+            "FromPort": 80,
+            "ToPort": 80,
+            "CidrIp": "0.0.0.0/0"
+          },
```

Comment:
Do we actually need to allow HTTP/80 from the whole internet? Won't all these requests just come from Route 53? Or is Route53 not a full proxy? I'm a bit shaky on how the ssl layer works.

Reply:
Doh, just realized cloudfront is doing this part for us. same question there though

Reply:
Route 53 is just Amazon's DNS control panel; my doc may have made that unclear.

But yeah, requests come through CloudFront. It appears permitting CloudFront-only traffic is tough, so I figure we can just leave this for now.